integration/inspektor-gadget/integration_test.go

```diff
@@ -1300,14 +1291,14 @@ func TestTcpconnect(t *testing.T) {
 				e.MountNsID = 0
 			}
 
-			return ExpectEntriesToMatch(output, normalize, expectedEntries...)
+			return ExpectEntriesToMatch(output, normalize, expectedEntry)
 		},
 	}
 
 	commands := []*Command{
 		CreateTestNamespaceCommand(ns),
 		tcpconnectCmd,
-		BusyboxPodRepeatCommand(ns, "wget -q -O /dev/null -T 3 http://1.1.1.1"),
+		BusyboxPodRepeatCommand(ns, "wget -q -O /dev/null -T 3 http://1.1.1.1:80"),
```

Comment:
We don't need to specify port here or you are trying to make it explicit?

Reply:
Just making things even more explicit. `http://` should be enough for port 80